template/pipeline-export.pl

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eux
```

Comment:
`.pl` ?

Reply:
A mistake, normally it is .tpl as in nbconvert.